woocommerce/woocommerce-admin
# Local docker setup for WC Admin

This PR adds Docker files to spin up local development for WC Admin

### Detailed test instructions:

I've tested this setup in Mac OS (Catalina) and Linux (Ubuntu). You should have Docker and Node installed to get started.

1. Clone the repository.
2. Run `npm docker-dev:config`. This will prompt you a few questions to set up your environment. For the purpose of this test, please hit enter to choose the default values.
3. Run the containers. `npm docker-dev:up`. 
4. Run `npm docker-dev:setup`. This runs `npm install`, `composer install`, and `npm run dev`. 
5. Once step #4 is finished, access http://localhost:8888.
